### PR TITLE
[KYUUBI #678]Fix: kyuubi ctl use default property if missing

### PR DESCRIPTION
**Pull request by hddong**
### _Why are the changes needed?_

when use `bin/kyuubi-ctl list` throw:
```
Exception in thread "main" org.apache.kyuubi.KyuubiException: Zookeeper namespace is not specified and no default value to load
        at org.apache.kyuubi.ctl.ServiceControlCliArguments.fail(ServiceControlCliArguments.scala:339)
        at org.apache.kyuubi.ctl.ServiceControlCliArguments.validateZkArguments(ServiceControlCliArguments.scala:136)
```
we expect: list engine nodes with default namespace `kyuubi`
### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.readthedocs.io/en/latest/tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by yaooqinn** [comment]
thank you very much @hddong, 
LGTM, only minor revisions.

**Comment by hddong** [comment]
@yaooqinn: thanks for your review, make some modifications to keep `verbose` log output.

**Review by yaooqinn — APPROVED** [review]

**Comment by yaooqinn** [comment]
LGTM

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ServiceControlCliArguments.scala [hunk 1]
```diff
@@ -63,11 +63,10 @@ class ServiceControlCliArguments(args: Seq[String], env: Map[String, String] = s
...
     if (action != ServiceControlAction.CREATE && namespace == null) {
-      conf.getOption(HA_ZK_NAMESPACE.key).foreach { v =>
-        if (verbose) {
-          info(s"Zookeeper namespace is not specified, use value from default conf:$v")
-        }
-        namespace = v
+      namespace = conf.get(HA_ZK_NAMESPACE)
+      if (verbose) {
+        info(s"Zookeeper namespace is not specified," +
+          s" use value from default conf:$namespace")
```
Comment (yaooqinn) [inline kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ServiceControlCliArguments.scala:69]: NOT related to the PR, can we avoid mess up verbose with log4j?

**Comment by yaooqinn** [comment]
thanks, merged to master

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ServiceControlCliArguments.scala (hunk 1 above)
Comment (hddong) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ServiceControlCliArguments.scala:69]: > NOT related to the PR, can we avoid mess up verbose with log4j?

IMO, we can have a try, use a custom log level.

Comment (yaooqinn) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/ServiceControlCliArguments.scala:69]: Yea
